compute/quickstart/compute_quickstart_sample_test.go

```diff
@@ -0,0 +1,180 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"reflect"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/integration/mtest"
+)
+
+var mt *mtest.T
+
+type mockManager struct{}
+
+func (m mockManager) load(t []trainer) error {
+	return nil
+}
+
+func (m mockManager) setCollection(c *mongo.Collection) {
+}
+
+func (m mockManager) list() ([]*trainer, error) {
+	trainers := []*trainer{
+		{"Ash", 10, "Pallet Town"},
+		{"Misty", 10, "Cerulean City"},
+		{"Brock", 15, "Pewter City"},
+	}
+
+	return trainers, nil
+}
+
+func newMockManager() mockManager {
```

Comment:
These two functions would be better just inlined since they're not doing any setup.

Reply:
yep. got rid of them.